style.css

```diff
@@ -0,0 +1,31 @@
+@import './Components/kube-ui.css';
+@import './docs-css/docs-nav.css';
+@import './docs-css/docs-sidebar.css';
+@import './docs-css/docs-code.css';
+@import './docs-css/docs-footer.css';
+@import './utility.css';
```

Comment:
Just a suggestion, in place of docs-CSS you can just use CSS or styles names as it provides more sense.

Reply:
I kept is as docs-css to represent "documentation css". I guess that make sense.

Reply:
Ok Cool!